### PR TITLE
Fix that timezone bug ... for now ...

### PR DESCRIPTION
**Pull request by mfernandez-rally**

### CODE REVIEW

File: src/app/components/calendar/week.js [hunk 1]
```diff
@@ -19,8 +19,9 @@ export default class Week extends React.Component {
     const daysBetween = (this.props.endDate.getTime() - this.props.startDate.getTime())/(1000 * 60 * 60 * 24);
     for (let i = 0; i <= Math.round(daysBetween); i++) {
       let date = new Date(this.props.startDate.getFullYear(), this.props.startDate.getMonth(), this.props.startDate.getDate() + i);
+      
       let scheduledRecipes = _.filter(this.props.scheduledRecipes, function(recipe) {
-        return recipe.scheduledDate === date.getTime();
+        return new Date(recipe.scheduledDate).toDateString() === date.toDateString();
```
Comment (WadePeterson) [inline src/app/components/calendar/week.js]: What about just using `startDate` and `endDate` props?

`return recipe.scheduleDate >= this.props.startDate.getTime() && recipe.scheduleDate < this.props.endDate.getTime()`

Comment (mfernandez-rally) [inline src/app/components/calendar/week.js]: This is parsing it out to each day component

Comment (WadePeterson) [inline src/app/components/calendar/week.js]: Yeah looks good, I get what it's doing now.

**Comment by WadePeterson** [comment]
🌴 🤕

**Pull request merged**